servatrice/src/serversocketinterface.cpp

```diff
@@ -1794,3 +1792,9 @@ void WebsocketServerSocketInterface::binaryMessageReceived(const QByteArray &mes
 
     processCommandContainer(newCommandContainer);
 }
+
+// TODO make this configurable?
+bool AbstractServerSocketInterface::isPasswordLongEnough(const int passwordLength)
+{
+    return passwordLength < 6;
```

Comment:
I'd like to see this configurable before merging, if possible <3

Reply:
Sure thing, hopefully it will be done by Monday.